src/Core/View.js

```diff
@@ -82,15 +114,30 @@ function _preprocessLayer(view, layer, provider) {
     // probably not the best place to do this
     if (layer.type == 'color') {
         layer.frozen = layer.frozen || false;
-        layer.visible = layer.visible == undefined ? true : layer.visible;
-        layer.opacity = layer.opacity == undefined ? 1.0 : layer.opacity;
-        layer.sequence = 0;
+        _defineLayerProperty(view, layer, 'visible', true);
```

Comment:
With this change a user can't declare a layer to be hidden by default. 
I'm fine with this but you need to warn the user that your ignoring/overriding `visible` and `opacity`